src/eei.cpp

```diff
@@ -790,6 +791,8 @@ inline int64_t maxCallGas(int64_t gas) {
 
       // NOTE: this must be declared outside the condition to ensure the memory doesn't go out of scope
       vector<uint8_t> contract_code;
+      HERA_DEBUG << "Taking GasSchedule::create" << endl;
+      takeInterfaceGas(GasSchedule::create);
```

Comment:
I don't think this is the correct place. Should be grouped with the other `takeInterfaceGas` call. Moved.